onnxruntime/core/mlas/lib/logistic.cpp

```diff
@@ -121,6 +121,11 @@ Return Value:
 
         float Value = *Input++;
 
+	// This odd "if test" exists to ensure an input value of NaN carries through
```

Comment:
>	 [](http://example.com/codeflow?start=0&length=1)

nit:space

Reply:
(see above)

Reply:
This line used a tab and the others used spaces.  When I edit it, it looks fine.
I will change it to use spaces as well.